x/rollappparams/types/genesis_test.go

```diff
@@ -45,33 +45,6 @@ func TestGenesisState(t *testing.T) {
 			},
 			expectedErr: true,
 		},
-		{
-			name: "block max gas too small",
-			params: func() types.Params {
-				p := types.DefaultParams()
-				p.Blockmaxgas = 0
-				return p
-			},
-			expectedErr: true,
-		},
-		{
-			name: "block max size too small",
```

Comment:
why did we remove the max size test?

Reply:
no reason for that, it was by mistake. tests readded